models/review-models.js

```diff
@@ -67,8 +67,11 @@ exports.getReviewByID = (inputId) => {
     return db
       .query(
```

Comment:
Looks good, I would suggest lies 64-66 are unnecessary as SQL will throw an error if the id is a string, would reccomend you use that error if you can.